CHANGELOG.md

```diff
@@ -3,6 +3,29 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.26
+
+_Mar 3, 2023_
+
+
+We're happy to announce the [v6 stable](https://github.com/mui/mui-x/releases/tag/v6.0.0) is finally available! 🎉
```

Comment:
Is it worth it to update to `/latest` to always land users on the latest release? (let's say a user visits this page when v6.x.y is the latest release but the link still redirects it to v6.0.0)
```suggestion
We're happy to announce the [v6 stable](https://github.com/mui/mui-x/releases/latest) is finally available! 🎉
```

Reply:
For me it makes sense to redirect to v6.0.0 instead of v6.x.y because it's the release not where v6 is introduced. The v6.x.y will only discuss the patch/bug fix

Reply:
Yep, makes sense 👍